source/config/featureFlagEnums.py

```diff
@@ -122,6 +122,23 @@ def _displayStringLabels(self):
 	NOTIFICATIONS = enum.auto()
 
 
+class FontAttributesBrailleModeFlag(DisplayStringEnum):
+	"""Enumeration containing the possible ways to display formatting changes in braille."""
+
+	DEFAULT = enum.auto()
+	LIBLOUIS = enum.auto()
+	TAGS = enum.auto()
+
+	@property
+	def _displayStringLabels(self) -> dict["FontAttributesBrailleModeFlag", str]:
+		return {
+			# Translators: The label for a braille mode
+			FontAttributesBrailleModeFlag.LIBLOUIS: _("Liblouis"),
```

Comment:
```suggestion
			FontAttributesBrailleModeFlag.LIBLOUIS: _("LibLouis"),
```

Reply:
The [Liblouis project](https://liblouis.io/) write it "Liblouis", so I don't see why we should deviate when it's their software.

Reply:
ah nevermind, thanks for pointing this out. I think we've been doing it the other way